File: public/phase-2-20160620022203/JAVA/a-and-b-and-compilation-errors.java
// Name : A and B and Compilation Errors
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the initial number of compilation errors
		// a : 
		// b : 
		// c : 
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);
		int[] b = new int[(n-1)];
		String[] b_elements = (in.nextLine()).split(" ");
		for(index=0;index<(n-1);index++)
			b[index] = Integer.parseInt(b_elements[index]);
		int[] c = new int[(n-2)];
		String[] c_elements = (in.nextLine()).split(" ");
		for(index=0;index<(n-2);index++)
			c[index] = Integer.parseInt(c_elements[index]);


		// write your code here
		// store your results in `errors_disappeared_after_first_correction`,`errors_disappeared_after_second_correction`

		// output
		// Dummy Data
		int errors_disappeared_after_first_correction=8;
		int errors_disappeared_after_second_correction=123;

		System.out.println(errors_disappeared_after_first_correction);
		System.out.println(errors_disappeared_after_second_correction);

	}
}

File: public/phase-2-20160620022203/JAVASCRIPT/a-and-b-and-compilation-errors.js
// Name : A and B and Compilation Errors
// input:
// n :  the initial number of compilation errors
// a : 
// b : 
// c : 
var index,errors_disappeared_after_first_correction,errors_disappeared_after_second_correction;
var n = parseInt(readline(), 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));
var b = [];
var b_elements = readline().split(' ');
for(index=0;index<(n-1);index++)
	b.push(parseInt(b_elements[index], 10));
var c = [];
var c_elements = readline().split(' ');
for(index=0;index<(n-2);index++)
	c.push(parseInt(c_elements[index], 10));


// write your code here
// store your results in `errors_disappeared_after_first_correction`,`errors_disappeared_after_second_correction`

// Dummy Data
errors_disappeared_after_first_correction = 8;
errors_disappeared_after_second_correction = 123;


// output
print(errors_disappeared_after_first_correction);
print(errors_disappeared_after_second_correction);
